sdk/monitor/monitor-ingestion/src/logsIngestionClient.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TokenCredential } from "@azure/core-auth";
+import { CommonClientOptions } from "@azure/core-client";
+import { GeneratedMonitorIngestionClient } from "./generated";
+import { UploadLogsError, UploadOptions, UploadResult, UploadStatus } from "./models";
+import { GZippingPolicy } from "./gZippingPolicy";
+import { asyncPool } from "./utils/concurrentPoolHelper";
+import { splitDataToChunks } from "./utils/splitDataToChunksHelper";
+
+/**
+ * Options for Montior Logs Ingestion Client
+ */
+export interface LogsIngestionClientOptions extends CommonClientOptions {
+  /** Api Version */
+  apiVersion?: string;
+}
+const defaultIngestionScope = "https://monitor.azure.com//.default";
```

Comment:
No this is the correct scope.

Reply:
Otherwise we'll get this error - The resource principal named <logs ingestion url> was not found in the tenant named Microsoft. This can happen if the application has not been installed by the administrator of the tenant or consented to by any user in the tenant. You might have sent your authentication request to the wrong tenant

Reply:
Wild! Apparently multiple slashes are syntactically valid according to the URL RFC. Thanks for confirming.